src/middleware/isAdmin.middleware.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Request, Response } from 'express';
+import { verify } from 'jsonwebtoken';
+import UserModel from '../models/user.model';
+
+export const isAdmin = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const token = req.headers['authorization'];
+  if (!token) return res.status(401).send('No User');
```

Comment:
It should be not `.send` it should be `.json` because then express will add all the required headers for json and it will be easier for frontend to handle json

You can also use `StatusCodes.UNAUTHORIZED` for status code it is much more to easier understand about status code